cardano-crypto-class/test/Test/Crypto/DSIGN.hs

```diff
@@ -3,7 +3,7 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-}
-
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
See https://github.com/input-output-hk/cardano-base/pull/107#discussion_r417109687 :grin:

Reply:
There are other instances of `{-# OPTIONS_GHC -fno-warn-orphans #-}` in this repo. I matched what is already there.

Reply:
I've swapped them all to the `-W` style in https://github.com/input-output-hk/cardano-base/pull/113